jetbrains-core/src/software/aws/toolkits/jetbrains/ui/ResourceSelector.kt

```diff
@@ -138,6 +138,8 @@ class ResourceSelector<T> : ComboBox<T>() {
         }
     }
 
+    fun values(): Set<T> = (0 until itemCount).map { getItemAt(it) }.toSet()
```

Comment:
If you switch out the model to a CollectionComboBoxModel you dont need this method

Reply:
But I guess if you want the set it doesnt save you

Reply:
Yes, switching to CollectionComboBoxModel gives me the List for free but since I want the Set, I'll keep it as is.

Reply:
nit: I think the `values()` method should still return a list - other resource selectors may care about ordering (and may contain duplicates) which you will lose if you move this to a `Set`.

Reply:
As a more long term comment I don't think we should be looking at the view ComboBox values for this at all - we should be looking at some canonical source of stacks - the combobox doesn't 'own' this data - the fact it's here is merely coincidental.

Soon we're going to need to have a cache of these sorts of things rather than calling down to the service all the time.